web/packages/teleport/src/services/audit/gen-event-reference/gen-event-reference.ts

```diff
@@ -0,0 +1,161 @@
+import { Event, Formatters } from '../types';
+
+// eventsWithoutExamples returns an array of event objects based on the
+// elements in formatters that do not have corresponding examples in fixtures.
+export function eventsWithoutExamples(
```

Comment:
This should be probably refactored in terms of `formatters.filter` instead of creating new object such as `result` just so that we can preserve types.

I mean the implementation could technically be kept as is, but you'd have to change `let result = []` to `let result: <actual type for what's in the array> []`.

From those two options, I think it's better to replace `Object.keys(formatters).forEach` with `formatters.filter`.